.github/workflows/pr.yml

```
@@ -0,0 +1,25 @@
+name: pr
+on: pull_request
```

this is the general pattern that i've seen in most repos

```suggestion
on:
  push:
    tags:
      - v*
    branches:
      - master
  pull_request:
```

---

Yep! I'll add releasing on the on push to master later.